Esri/arcgis-maps-sdk-kotlin-samples
# Geodatabase replica from feature service

## Description
PR to add a new Kotlin sample _"Generate Geodatabase Replica From Feature Service"_ in `Edit and Manage Data` category.

## Links and Data
Sample Epic: `runtime/kotlin/issues/1802`

## What To Review
-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files

## How to Test
- Run the sample on the sample viewer or the repo.
- Zoom to any extent and click the generate button. A red outline will show the extent used. 
- The job's progress is shown while the geodatabase is generated. When complete, the map will reload with only the layers in the geodatabase, clipped to the extent.
